index.html

```diff
@@ -2,6 +2,7 @@
 <DOCTYPE! html>
   <head>
      <title> first html and css project</title>
+     <link rel="stylesheet" href="style.css">
   </head>
   <body>
   Hello World,We are software developers! 
```

Comment:
please,  it would be better to use the h1 tag  in the body text

Reply:
Kindly explain why he should do that. Like you can say to him that this is a headline so it's more semantic to put the text inside a header tag like `h1`. So he knows why I should do that.